com.salesforce.b2eclipse.jdt.ls/src/main/java/com/salesforce/b2eclipse/command/BazelWorkspaceCommandRunner.java

```diff
@@ -466,6 +509,19 @@ public void runBazelVersionCheck() throws BazelCommandLineToolConfigurationExcep
         }
     }
 
+    public String getOperatingSystemDirectoryName() {
+        if (SystemUtils.IS_OS_MAC) {
+            return "darwin";
+        }
+        if (SystemUtils.IS_OS_WINDOWS) {
+            return "windows";
+        }
+        if (SystemUtils.IS_OS_LINUX) {
+            return "linux";
+        }
+        return null;
```

Comment:
maybe throw exception or return empty string to avoid NPE in case of unexpected OS?